config.py

```diff
@@ -825,6 +780,26 @@
 Whether independent component analysis should be used or not.
 """
 
+ica_reject: Optional[Dict[str, float]] = None
+"""
+Peak-to-peak amplitude limits to exclude epochs from ICA fitting.
+
+This allows you to remove strong transient artifacts, which could negatively
+affect ICA performance.
+
+The BIDS Pipeline will automatically try to detect EOG and ECG artifacts in
+your data, and remove them. For this to work properly, it is recommended
+to **not** specify rejection thresholds for EOG and ECG channels here –
+otherwise, ICA won't be able to "see" these artifacts.
+
+???+ example "Example"
+    ```python
+    reject = {'grad': 4000e-13, 'mag': 4e-12, 'eeg': 200e-6}
+    reject = {'grad': 4000e-13}
+    reject = None
```

Comment:
You mean ica_reject

Reply:
Also you should recommend much higher values than for reject

Reply:
Better now?